presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedQueriesWithExchangeMaterialization.java

```diff
@@ -42,9 +44,18 @@ public void testMaterializedExchangesEnabled()
 
     @Test
     public void testMaterializeHiveUnsupportedTypeForTemporaryTable()
+    {
+        testMaterializeHiveUnsupportedTypeForTemporaryTable(ORC, true);
+        testMaterializeHiveUnsupportedTypeForTemporaryTable(PAGEFILE, false);
```

Comment:
Should this also test the failure cases, IOWs to validate that the test will fail if these things don't work also test with ORC and false.